src/migration/1740042768350-UserSubscriptionProvider.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserSubscriptionProvider1740042768350 implements MigrationInterface {
+  name = 'UserSubscriptionProvider1740042768350'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`UPDATE "user"
+      SET "subscriptionFlags" = jsonb_set("subscriptionFlags", '{provider}', '"paddle"')
+      WHERE "subscriptionFlags" <> '{}'::jsonb;
+    `);
```

Comment:
Let's run this as a one of, don't think we need a migration

Reply:
Reverted. Will run manually